.github/dependabot.yml

```diff
@@ -56,3 +56,7 @@ updates:
     directory: "images/benthos-v4/context/"
     schedule:
       interval: "daily"
+  - package-ecosystem: "docker"
+    directory: "images/techdocs-cli/context/"
+    schedule:
+      interval: "daily"
```

Comment:
We should add npm here for `images/techdocs-cli/context`, and I'm also not sure if python works recursively, I suspect it does not, so I think we should add an entry for that also.

Reply:
fixed

Reply:
fixed